### PR TITLE
fix(webpack): can't resolve absolute url in css

### LINKED ISSUE #8508: Build error with nuxt-edge 2.14.13 - Can't resolve a CSS @font-face url.
Using `"nuxt-edge": "^2.14.13-26803446.5318172",` my build fails with the below error. Was working fine with latest `nuxt` version previously. 

My nuxt config has this:

```
    css: [
        "~/styles/css-variables.scss",
        "~/styles/global.scss",
        "~/styles/transitions.scss",
        "~/styles/fonts.css",
    ],
```

```

 ERROR  in ./styles/fonts.css                                                               friendly-errors 13:37:43


Module build failed (from ./node_modules/css-loader/dist/cjs.js):                           friendly-errors 13:37:43
Error: Can't resolve '/fonts/AVENIRNEXTLTPRO-CN.eot' in '/Users/drew/Sites/loganindustry-frontend/styles'
    at /Users/drew/Sites/loganindustry-frontend/node_modules/enhanced-resolve/lib/Resolver.js:209:21
    at /Users/drew/Sites/loganindustry-frontend/node_modules/enhanced-resolve/lib/Resolver.js:285:5
    at eval (eval at create (/Users/drew/Sites/loganindustry-frontend/node_modules/tapable/lib/HookCodeFactory.js:33:1
0), <anonymous>:15:1)
    at /Users/drew/Sites/loganindustry-frontend/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:44:7
    at /Users/drew/Sites/loganindustry-frontend/node_modules/enhanced-resolve/lib/Resolver.js:285:5
    at eval (eval at create (/Users/drew/Sites/loganindustry-frontend/node_modules/tapable/lib/HookCodeFactory.js:33:1
0), <anonymous>:15:1)
    at /Users/drew/Sites/loganindustry-frontend/node_modules/enhanced-resolve/lib/Resolver.js:285:5
    at eval (eval at create (/Users/drew/Sites/loganindustry-frontend/node_modules/tapable/lib/HookCodeFactory.js:33:1
0), <anonymous>:27:1)
    at /Users/drew/Sites/loganindustry-frontend/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:67:43
    at /Users/drew/Sites/loganindustry-frontend/node_modules/enhanced-resolve/lib/Resolver.js:285:5
    at eval (eval at create (/Users/drew/Sites/loganindustry-frontend/node_modules/tapable/lib/HookCodeFactory.js:33:1
0), <anonymous>:16:1)
    at /Users/drew/Sites/loganindustry-frontend/node_modules/enhanced-resolve/lib/Resolver.js:285:5
    at eval (eval at create (/Users/drew/Sites/loganindustry-frontend/node_modules/tapable/lib/HookCodeFactory.js:33:1
0), <anonymous>:16:1)
    at /Users/drew/Sites/loganindustry-frontend/node_modules/enhanced-resolve/lib/Resolver.js:285:5
    at eval (eval at create (/Users/drew/Sites/loganindustry-frontend/node_modules/tapable/lib/HookCodeFactory.js:33:1
0), <anonymous>:16:1)
    at /Users/drew/Sites/loganindustry-frontend/node_modules/enhanced-resolve/lib/Resolver.js:285:5
                                                                                            friendly-errors 13:37:43
 @ ./styles/fonts.css 4:14-157 14:3-18:5 15:22-165
 @ ./.nuxt/App.js
 @ ./.nuxt/index.js
 @ ./.nuxt/client.js
 @ multi eventsource-polyfill webpack-hot-middleware/client?reload=true&timeout=30000&ansiColors=&overlayStyles=&path=
%2F__webpack_hmr%2Fclient&name=client ./.nuxt/client.js
```

It's my first time using `nuxt-edge` so I'm not sure if I'm doing it right, but here is my package dependencies if that helps:

```
"dependencies": {
        "@nuxtjs/dotenv": "^1.4.1",
        "@vimeo/player": "^2.14.1",
        "cheerio": "^1.0.0-rc.3",
        "core-js": "^3.6.5",
        "intrinsic-scale": "^3.0.4",
        "lodash": "^4.17.20",
        "nuxt-edge": "^2.14.13-26803446.5318172",
        "youtube-player": "^5.5.2"
    },
    "devDependencies": {
        "@babel/plugin-proposal-object-rest-spread": "^7.11.0",
        "@nuxtjs/sitemap": "^2.4.0",
        "@nuxtjs/storybook": "^2.2.1",
        "@nuxtjs/style-resources": "^1.0.0",
        "eslint": "^7.9.0",
        "eslint-plugin-vue": "^6.2.2",
        "graphql": "^15.3.0",
        "graphql-tag": "^2.11.0",
        "node-sass": "^4.14.1",
        "nodemon": "^2.0.4",
        "nuxt-graphql-request": "^3.1.1",
        "prettier": "^2.1.2",
        "prettier-eslint": "^11.0.0",
        "sass-loader": "^9.0.3",
        "vue-svg-loader": "^0.16.0",
        "vue-template-compiler": "^2.6.12",
        "webfontloader": "^1.6.28"
    }
```

### PR DESCRIPTION
**Pull request by clarkdo**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

Fix https://github.com/nuxt/nuxt.js/issues/8508

Latest css-loder won't resolve absolute url now, but we'll see can't resolve pth error from webpack, this pr will ignore absolute path (will cover by serve-static or cdn path) and print a warning message.

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/webpack/src/utils/style-loader.js [hunk 1]
```diff
@@ -25,6 +26,16 @@ export default class StyleLoader {
     return Boolean(this.isServer && this.extractCSS)
   }
 
+  isUrlResolvingEnabled (url, resourcePath) {
+    // Ignore absolute URLs, it will be handled by serve-static.
+    if (url.startsWith('/')) {
+      consola.warn(`Please use relative path or alias path instead of absolute path ${url}`)
```
Comment (clarkdo) [inline packages/webpack/src/utils/style-loader.js:32]: Absolute url works when we ignore it from css-loader, but I think we'd better put the warning message here for letting user use relative or alias path

**Review by pi0 — DISMISSED** [review]

**Comment by husayt** [comment]
@clarkdo @pi0 the problem with this approach that it forces using `@/static/...` in css, which results in url starting with"/_nuxt/...".  I want to use "http://www.example.com/fonts/..." but this ends up like "http://www.example.com/_nuxt/fonts/...".

**Comment by clarkdo** [comment]
I’ll have a look tmr, but I don’t think you should use @/static, you can use relative path ‘../static/...’ or absolute path ‘/static/...’

**Comment by husayt** [comment]
That is what I was using before "/fonts/xyz.woff". But since this was merged it gives a warning `Please use relative path or alias path instead of absolute path `

**Comment by clarkdo** [comment]
Have you tried ‘~static/..’, latest css-loader requires ~ as prefix of alias.

**Comment by husayt** [comment]
Yes, but that is the problem ‘~static/fonts/..’ gets replaced by "/_nuxt/fonts/..."

**Comment by clarkdo** [comment]
@husayt Sorry, `~static` as alias will create a wrapper js module and bundled into dist files, so `"/_nuxt/fonts/..."` works, I think what you want is using `/fonts/..` without bundling fonts, it makes sense, I'll remove the warning message. Can you please double confirm it works OK except current warning message ?

**Comment by husayt** [comment]
Exactly. Once change is commited I will check on edge and confirm.

**Pull request merged**